docs/3.0.0-beta.x/getting-started/contributing.md

```diff
@@ -16,6 +16,15 @@ This repository contains the core of Strapi. It means the admin panel, core plug
 
 This is the code used to generate the official Docker images for Strapi. These images are available through our [Docker Hub](https://hub.docker.com/r/strapi/strapi).
 
+## Request for Comments (RFC)
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are “substantial”, and we ask that these be put through a bit of a design process and produce a consensus among the Strapi core team.
```

Comment:
Some changes though, are "substantials" and we ask that they be put through a bit of desgin process to produce a consensus among the Strapi core team.

Reply:
As this sentence comes from the [React doc](https://reactjs.org/docs/how-to-contribute.html#request-for-comments-rfc) directly I suggest we trust their english instead of mine!